design/BulkAPI.md

```diff
@@ -114,73 +114,68 @@ When verbose=true, additional detailed information about the job is provided.
+      "notification": {},
+      "recommendation": {
+        "status": "unprocessed",
+        "notification": {}
+      }
+    },
+    {
+      "name": "prometheus-1|default|kube-system|kindnet(deployment)|kindnet-cni",
+      "notification": {},
+      "recommendation": {
+        "status": "processed",
+        "notification": {}
+      }
     },
-    "recommendations": {
-      "data": {
-        "processed": [
-          "prometheus-1|default|monitoring|alertmanager-main(statefulset)|config-reloader",
-          "prometheus-1|default|monitoring|node-exporter(daemonset)|node-exporter",
-          "prometheus-1|default|local-path-storage|local-path-provisioner(deployment)|local-path-provisioner",
-          "prometheus-1|default|monitoring|alertmanager-main(statefulset)|alertmanager",
-          "prometheus-1|default|monitoring|prometheus-operator(deployment)|kube-rbac-proxy",
-          "prometheus-1|default|kube-system|coredns(deployment)|coredns",
-          "prometheus-1|default|monitoring|blackbox-exporter(deployment)|kube-rbac-proxy",
-          "prometheus-1|default|monitoring|prometheus-k8s(statefulset)|config-reloader",
-          "prometheus-1|default|monitoring|prometheus-operator(deployment)|prometheus-operator",
-          "prometheus-1|default|monitoring|node-exporter(daemonset)|kube-rbac-proxy",
-          "prometheus-1|default|monitoring|kube-state-metrics(deployment)|kube-rbac-proxy-self",
-          "prometheus-1|default|monitoring|kube-state-metrics(deployment)|kube-state-metrics",
-          "prometheus-1|default|monitoring|kruize(deployment)|kruize",
-          "prometheus-1|default|monitoring|blackbox-exporter(deployment)|module-configmap-reloader",
-          "prometheus-1|default|monitoring|prometheus-k8s(statefulset)|prometheus",
-          "prometheus-1|default|monitoring|kube-state-metrics(deployment)|kube-rbac-proxy-main",
-          "prometheus-1|default|kube-system|kube-proxy(daemonset)|kube-proxy",
-          "prometheus-1|default|monitoring|prometheus-adapter(deployment)|prometheus-adapter",
-          "prometheus-1|default|monitoring|grafana(deployment)|grafana",
-          "prometheus-1|default|kube-system|kindnet(daemonset)|kindnet-cni",
-          "prometheus-1|default|monitoring|kruize-db-deployment(deployment)|kruize-db",
-          "prometheus-1|default|monitoring|blackbox-exporter(deployment)|blackbox-exporter"
-        ],
-        "processing": [
-          "prometheus-1|default|cadvisor|cadvisor(daemonset)|cadvisor"
-        ],
-        "unprocessed": [
-        ],
-        "failed": []
+    {
+      "name": "prometheus-1|default|monitoring|kruize(deployment)|kruize",
+      "notification": {},
+      "recommendation": {
+        "status": "processing",
+        "notification": {}
+      }
+    },
+    {
+      "name": "prometheus-1|default|monitoring|kruize(deployment)|kruize",
+      "recommendation": {
+        "status": "failed",
+        "notifications": {
```

Comment:
I see `notifications` in some places and `notification` in some, lets keep it as `response` in all places

Reply:
I changed it to  `notifications` similar to updateRecommendations response .